minecraft_servers/apfelnetzwerk/manifest.json

```diff
@@ -0,0 +1,81 @@
+{
+  "server_name": "apfelnetzwerk",
+  "nice_name": "Apfelnetzwerk",
+  "direct_ip": "play.apfelnetzwerk.de",
+  "server_wildcards": [
+    "%.apfelnetzwerk.de"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://apfelnetzwerk.de/",
+    "web_shop": "https://shop.apfelnetzwerk.de/",
+    "web_support": "https://support.apfelnetzwerk.de/",
+    "twitter": "Apfelnetzwerk",
+    "instagram": "Apfelnetzwerk",
+    "discord": "https://discord.gg/apfelnetzwerk"
+  },
+  "gamemodes": {
+    "BedWars": {
+      "name": "BedWars",
+      "color": "#d51212",
+      "url": "https://apfelnetzwerk.de/spiele/bedwars.html"
+    },
+    "SkyWars": {
+      "name": "SkyWars",
+      "color": "#00e9fc",
+      "url": "https://apfelnetzwerk.de/spiele/skywars.html"
+    },
+    "TTT": {
+      "name": "TTT",
+      "color": " #9f0101 ",
+      "url": "https://apfelnetzwerk.de/spiele/ttt.html"
+    },
+    "TryJump": {
+      "name": "TryJump",
+      "color": "#0673cd",
+      "url": "https://apfelnetzwerk.de/spiele/tryjump.html"
+    },
+    "FreeBuild": {
+      "name": "FreeBuild",
+      "color": "#34e300",
+      "url": "https://apfelnetzwerk.de/spiele/freebuild.html"
+    },
+    "Training": {
+      "name": "Training",
+      "color": "#ff6e26",
+      "url": "https://apfelnetzwerk.de/spiele/Bridge.html"
+    },
+    "UHC": {
+      "name": "UHC",
+      "color": "#dad000",
+      "url": "https://apfelnetzwerk.de/spiele/uhc.html"
+    },
+    "1vs1": {
+      "name": "1vs1",
+      "color": "#ffffff",
+      "url": "https://apfelnetzwerk.de/spiele/1vs1.html"
+    },
+    "KitPvP": {
+      "name": "KitPvP",
+      "color": "#00ffbd"
+    },
+    "Target": {
+      "name": "Target",
+      "color": "#ff6060"
+    },
+    "Community": {
+      "name": "Community",
+      "color": "#c600a5"
+    }
+  },
+  "discord": {
+    "server_id": "851012884674641973"
```

Comment:
Remove quotes, as it is the case in our example. Currently, it is a string containing your server id.
https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#example-manifestjson-file